### PR TITLE
Added an image for igblast

### PR DESCRIPTION
**Pull request by boratyng**
Dockerfile and scripts in `igblast` directory.

### CODE REVIEW

File: igblast/README.md
Comment (christiam) [inline igblast/README.md:1]: Is this file deliberately empty?

**Comment by christiam** [comment]
May I suggest modeling this directory after the contents of https://github.com/ncbi/docker/tree/master/sra-toolkit ?

File: igblast/README.md
Comment (jianye00) [reply igblast/README.md:1]: I need to add something to that file, I'll try later today.

**Comment by boratyng** [comment]
> May I suggest modeling this directory after the contents of https://github.com/ncbi/docker/tree/master/sra-toolkit ?

@christiam, what do you mean exactly? I see that we can do the Makefile with similar targets, anything else?

**Comment by christiam** [comment]
> > May I suggest modeling this directory after the contents of https://github.com/ncbi/docker/tree/master/sra-toolkit ?
> 
> @christiam, what do you mean exactly? I see that we can do the Makefile with similar targets, anything else?

I think that replacing the `VERSION` and `build-image.sh` files with a single Makefile and instructions on the `README.md` file will make this directory easier to maintain and use.

File: igblast/README.md
Comment (jianye00) [reply igblast/README.md:1]: I have finished readme file now.

**Comment by christiam** [comment]
I added a small change to the README.md file, see #31

**Review by christiam — APPROVED** [review]

**Pull request merged**